mobile_app/en/strings.json

```
@@ -1,109 +1,124 @@
+    "cancelButton": "cancel",
+    "saveButton": "save",
+    "gallery": "Gallery",
+    "camera": "Camera",
+    "uidCopy": "ID (IPv6) copied to clipboard",
+    "language": "Language",
+    "qrText": "Friends can scan this QR to add you as contact",
+    "textBackup": "Enter your PIN code to continue",
+    "erroPin": "Wrong PIN",
+    "createBackup": "Create backup file"
+  },
+  "Contacts": {
+    "addContact": "Add contact",
+    "editContact": "Edit contact",
+    "name": "Nickname",
+    "uid": "IPv6 address",
+    "wait": "please wait",
+    "existName": "there is already a contact with that nickname",
+    "existContact": "existing contact",
+    "invalidFormat": "Invalid format",
+    "contactSuccessfully": "Contact created successfully!"
+  },
+  "Header": {
+    "contacts": "Contacts",
+    "settings": "Settings",
+    "search": "Search",
+    "notConnected": "Not connected"
+  },
+  "Chats": {
+    "message": "Message",
+    "cancelAudio": "Slide to cancel",
+    "commentImage": "Add comment",
+    "titleDelete": "Delete chat",
+    "deleteBody": "Are you sure you want to delete all messages?",
+    "clean": "Clear chat",
+    "voice": "Voice",
+    "copyingMessage": "Message copied to clipboard",
+    "photo": "Photo"
+  },
+  "Languages": {
+    "en": "English",
+    "es": "Spanish"
+  },
+  "DeviceSettings": {
+    "retryButton": "retry",
+    "headerTitle": "device settings",
+    "errorInfo": "Connection failed. Please verify that you are connected to a Turpial device's WiFi hotspot.",
+    "deviceTitle": "Device Characteristics",
+    "bateryStatus": "Battery status",
+    "devicename": "Device name",
+    "compilationVersion": "Compilation version",
+    "availableMemory": "Available memory",
+    "wapTitle": "WAP (WiFi Access Point)",
+    "changePassword": "Change password",
+    "wstTitle": "WST (WiFi Station)",
+    "active": "Active",
+    "changeNameWap": "Change WAP name",
+    "placeholderWap": "Change name",
+    "changePasswordWap": "Change WAP password",
+    "placeholderPWapPassword": "Change password",
+    "changeNameSta": "Change STA name",
+    "placeholderSta": "Change name",
+    "changePasswordSta": "Change STA password",
+    "placeholderPStaPassword": "Change password",
+    "credentialTitle": "Credentials",
+    "ChangeCredencial": "Add new Credentials",
+    "ChangePassword": "Change password",
+    "usernameError": "This field can't be empty",
+    "passwordError": "This field can't be empty",
+    "confirmPassword": "Passwords don't match"
+  },
+  "WifiConnect": {
+    "title": "connect to a valid AP",
+    "subtitle": "This application to work correctly it is necessary to connect to a network compatible with ipv6, please connect to a valid network.",
+    "buttonConnect": "Connect",
+    "buttonCancel": "Cancel",
+    "connectionError": "connection error",
+    "successfulConnection": "successful connection",
+    "inputSSID": "Enter SSID",
+    "inputPasword": "Enter Password"
+  }
+}
```

New Line at the end of file
```suggestion
}

```